updating/preparing_for_updates/updating-cluster-prepare.adoc

```diff
@@ -27,15 +27,11 @@ Without the correct micro-architecture requirements, the update process will fai
 [id="kube-api-removals_{context}"]
 == Kubernetes API removals
 
-There are no Kubernetes API removals in {product-title} 4.15.
+{product-title} 4.16 uses Kubernetes 1.29, which removed a deprecated API.
 
-// Commenting out this section because there are no APIs being removed in OCP 4.15 / Kube 1.28. But we'll need this section again for 4.16
-////
-{product-title} 4.14 uses Kubernetes 1.27, which removed several deprecated APIs.
-
-A cluster administrator must provide a manual acknowledgment before the cluster can be updated from {product-title} 4.13 to 4.14. This is to help prevent issues after upgrading to {product-title} 4.14, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
+A cluster administrator must provide a manual acknowledgment before the cluster can be updated from {product-title} 4.15 to 4.16. This is to help prevent issues after upgrading to {product-title} 4.16, where APIs that have been removed are still in use by workloads, tools, or other components running on or interacting with the cluster. Administrators must evaluate their cluster for any APIs in use that will be removed and migrate the affected components to use the appropriate new API version. After this evaluation and migration is complete, the administrator can provide the acknowledgment.
 
-Before you can update your {product-title} 4.13 cluster to 4.14, you must provide the administrator acknowledgment.
+Before you can update your {product-title} 4.15 cluster to 4.16, you must provide the administrator acknowledgment.
```

Comment:
Not sure if this sentence needs to be here given the paragraph above?

Reply:
Good point, it is a bit repetitive. We have had this in all past releases like this... I'll think on whether to remove it or just leave it as is for consistency. Thanks!